src/System.Management.Automation/engine/parser/TypeInferenceVisitor.cs

```diff
@@ -174,6 +175,14 @@ internal IList<object> GetMembersByInferredType(PSTypeName typename, bool isStat
             List<object> results = new List<object>();
```

Comment:
Any reason why this is `new Linux<PSMemberInfo>`?

Reply:
Not that I can think of - I'll make the change and see what happens :)

Reply:
I was wrong. It needs to be object, since we have expressions like
`results.AddRange(cimClass.CimClassProperties);`